client_receiver.py

```diff
@@ -28,17 +28,17 @@ def window_draw(self):
                 # Waiting for message from server and print them
                 async for msg in ws:
                     # Cycle "for" is finished when client disconnects from server
-                    try:
-                        message = msg.json(loads=json.loads)
-                        self.state = State.from_dict(message)
+                    message = msg.json(loads=json.loads)
+                    if message["game_state"]:
+                        self.state = State.from_dict(message["game_state"])
```

Comment:
Proč je tady to "game_state"? nějak ten zápis nechápu, proč se to nenechá jen 
```
message = msg.json(loads=json.loads)
if message:
         self.state = State.from_dict(message["game_state"])
```

Reply:
No vidíš, o tom jsem se nezmínila. Server posílá několik různých informací, navíc opakovaně, proto by bylo dobře, kdyby se každá zpráva jednoznačně "představila" a řekla, jaký druh informací posílá, např. server pošle karty, ze kterých hráč může vybrat. Ale taky pokud si hráč karty nevybere, server pošle hráči automaticky vybrané karty. Jak klient má vědět, které karty jsou ty k vybraní, a které jsou ty přiřazené? (V obou případech pošleme seznam). Odtud pramení přidání jedné vrstvy nad syrová data - v backendu to vypadá takto:
```python
def as_dict(self, map_name):
        return {"game_state": {
                "board": get_map_data(map_name),
                "robots": [robot.as_dict() for robot in self.robots], }}
```

Reply:
Aha dobře chápu. No protože je to client receiver, tak mi to přišlo fuk, ale v souvislosti s hráčem je to dobrý.